extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -106,6 +106,10 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 		await view.initializeModel(form.component());
 	}
 
+	private createIconTextCell(icon: IconPath, text: string): string {
```

Comment:
Why are you making a separate function when it just returns the string directly?